html/captive-portal/templates/release.html

```diff
@@ -17,17 +18,31 @@
   // require access to web notifications now so it's ready when the access is activated
   initWebNotifications();
 });
-
 </script>
 
-<div id="about">
-  <img src="/content/images/unlock.png" alt="Access granted" />
-  <p id="toReplace">[% i18n("Your network access is currently being enabled. Once network connectivity is established you will be automatically redirected.") %]</p>
-  <p id="browser_notes"></p>
-  <script type="text/javascript" src="/content/timerbar.js"></script>
-  <noscript><p>[% i18n("If you have scripting turned off, you will not be automatically redirected. Please enable scripting or open a new browser window from time to time to see if your access was enabled.") %]</p></noscript>
+<div id="detectionNotice" class="media media--notice u-p u-mb">
+  <div class="media__img">[% flashIcon(level='notice') %]</div>
+  <p class="media__body">[% i18n("Your network access is currently being enabled. Once network connectivity is established you will be automatically redirected.") %]</p>
+</div>
+
+<div id="detectionError" class="media media--error u-p u-mb" style="display: none">
+  <div class="media__img">[% flashIcon(level='error') %]</div>
+  <p class="media__body">[% i18n("Unable to detect network connectivity. Try restarting your web browser or opening a new tab to see if your access has been succesfully enabled.") %]</p>
 </div>
 
+<div class="layout layout--auto">
+  <div class="layout__item">
+    <script type="text/javascript" src="/content/timerbar.js"></script>
```

Comment:
Timerbar is a bit off and not too in style with the new portal

![image](https://cloud.githubusercontent.com/assets/3857942/14050826/da464eea-f294-11e5-8eb3-d8a980c9ef26.png)